docs/05_components/notifications/notifications-ui/01_initial-configuration.mdx

```diff
@@ -161,4 +153,66 @@ When you generate a blank project, it puts the Inbox inside a flyout using the f
 </zero-flyout>
 ```
 
-Check the [generated project with the pbc-notify-seed](#on-a-new-application) for more details.
\ No newline at end of file
+### Foundation Router
+
+```json
+"@genesislcap/foundation-ui": "^14.0.0",
+"@genesislcap/web-core": "^14.0.0",
+```
+
+```ts
+// components.ts
+import { FoundationRouter } from '@genesislcap/foundation-ui';

```

Comment:
Can we add a note saying

"Add the foundation router to your `components.ts` file to register the foundation router web component"

```
...
import { FoundationRouter } from '@genesislcap/foundation-ui';
...
FoundationRouter;
...
```

Reply:
And the dots to indicate you might have code already